Tasks/DockerV2/task.json

```diff
@@ -108,6 +108,14 @@
             "label": "Arguments",
             "groupName": "commands",
             "helpMarkDown": "Docker command options. Ex:<br> For build command,<br>--build-arg HTTP_PROXY=http://10.20.30.2:1234 --quiet"
+        },
+        {
+            "name": "hidePII",
+            "type": "boolean",
+            "label": "Hide identifiable information from image metadata",
+            "groupName": "commands",
+            "defaultValue": false,
+            "helpMarkDown": "Prevent publishing identifiable information to image metadata store"
```

Comment:
By default pipeline data like source branch name, build id are added which helps with trace-ability. For example you can inspect an image to find out which pipeline built the image. You can opt out of this default behavior by using this input.